.ci/deps.sh

```diff
@@ -89,6 +89,11 @@ if [ "x${PLUGIN}" == "xmodel/tensorflow_hub" ]; then
   python -m pip install -U -e "./model/tensorflow"
 fi
 
+if [ "x${PLUGIN}" == "xmodel/autosklearn" ]; then
```

Comment:
The pattern we follow here is to install the dependencies if we are running just the tests for the model, `if [[ "x${PLUGIN}" == "xmodel/autosklearn" ]]` or the main package `[[ "x${PLUGIN}" == "x." ]]` or the docs `[[ "x${PLUGIN}" == "xdocs" ]]`

This is because the main package will run `dffml service dev install`, which installs all plugins in development mode. And the docs will too, since we need to have it installed to check which models it provides so we can auto generate the entries on the models plugin page.

```suggestion
if [[ "x${PLUGIN}" == "xmodel/autosklearn" ]] || \
   [[ "x${PLUGIN}" == "x." ]] || \
   [[ "x${PLUGIN}" == "xdocs" ]]; then
```

Reply:
Thanks